Source/SteveSloth/Private/MyEnemyAIDeadState.cpp

```diff
@@ -0,0 +1,34 @@
+/****************************************************************************************
+ * Copyright:
+ * Name: Guy Lapensee
+ * Script: ItemBaseClass
+ * Date: April 29, 2024
+ * Description: Enemy AI dead state for enemy base class
+ * TODO:
+ * Known Bugs:
+ ****************************************************************************************/
+
+
+#include "MyEnemyAIDeadState.h"
+
+MyEnemyAIDeadState::MyEnemyAIDeadState()
+{
+	isDead = false;
+}
+
+MyEnemyAIDeadState::~MyEnemyAIDeadState()
+{
+}
+
+void MyEnemyAIDeadState::DeadAnimation()
+{
+	// PLAY DEATH ANIMATION
+	// WAIT X TIME FOR DESTUCT ACTOR
+	// DROP ITEM()
+}
+
+void MyEnemyAIDeadState::DropItem()
```

Comment:
1: This function is never called.
2: this function should chose from a list of items that the specific enemy inheriting this class will drop.
3: then it should instantiate that item in the game at the location of the enemy.
Again these are some things that i would put in here.